sdk/python/setup.py

```diff
@@ -46,7 +46,7 @@
 REQUIRES_PYTHON = ">=3.7.0"
 
 REQUIRED = [
-    "click>=7.0.0",
+    "click>=7.0.0,<8.0.2",
```

Comment:
Why was this pinned to `<8.0.2`?

Reply:
I pinned this because click 8.1.0 causes https://github.com/psf/black/issues/2964